examples/bbm_bbm_1d/bbm_bbm_1d_basic.jl

```diff
@@ -4,7 +4,8 @@ using DispersiveShallowWater
 ###############################################################################
 # Semidiscretization of the BBM-BBM equations
 
-equations = BBMBBMEquations1D(gravity_constant = 9.81, D = 2.0)
+bathymetry_type = bathymetry_flat # bathymetry_variable
+equations = BBMBBMEquations1D(bathymetry_type, gravity_constant = 9.81)
```

Comment:
If we always use this pattern, should we make the bathymetry type a keyword argument?

Reply:
I did that in 72e0e40df700b9ab77e7f51ee01d5d89c91501c1.